src/group_theory/abelianization.lean

```diff
@@ -107,3 +113,37 @@ theorem hom_ext (φ ψ : abelianization G →* A)
 monoid_hom.ext $ λ x, quotient_group.induction_on x $ monoid_hom.congr_fun h
 
 end abelianization
+
+/-- Equivalent groups have equivalent abelianizations -/
+def mul_equiv.abelianization_congr {G H : Type*} [group G] [group H] (e : G ≃* H) :
```

Comment:
Can you add a `variables {G H : Type*} [group G] [group H]` line to make all these lemmas shorter?

Reply:
(or `G₁` and `G₂` to make `trans` match the others)

Reply:
I’m getting better at mind reading (but not good enough to do it before I press the “request review” button… sorry for that)